### PR TITLE
[Intel MKL] Graph Transform tool fold_old_batchnorms biasadd node bug

### PR DESCRIPTION
**Pull request by nammbash**
tensorflow/tensorflow/tools/graph_transforms/fold_old_batch_norms.cc has a bug.

Biasadd node copy needs to copy the attribute dataformat. Else there will be a load bug after batchnorm folding.. i.e. The data_format attribute of NCHW is not copied from conv2d node to Biasadd node without which the shape for convolution with the mismatched order is improperly checked.

### CODE REVIEW

**Review by drpngx — APPROVED** [review]

**Comment by drpngx** [comment]
If you can think of a test to add, please add it.
